cadc-access-control-admin/src/main/java/ca/nrc/cadc/ac/admin/ViewUser.java

```diff
@@ -105,12 +105,25 @@ protected void execute()
             log.debug("principal: " + this.getPrincipal());
             User user = this.getUserPersistence().getUser(this.getPrincipal());
             this.printUser(user);
+            this.systemOut.println("User Status: ACTIVE");
         }
         catch (AccessControlException | UserNotFoundException e)
         {
-            // Not in the main tree, try the pending tree
-            User user = this.getUserPersistence().getUserRequest(this.getPrincipal());
-            this.printUser(user);
+            try
+            {
+                // Not in the main tree, try the pending tree
+                User user = this.getUserPersistence().getUserRequest(this.getPrincipal());
+                this.printUser(user);
+                this.systemOut.println("User Status: PENDING");
+            }
+            catch (AccessControlException | UserNotFoundException ue)
+            {
+                // Check to see if it's locked
+                log.debug("principal: " + this.getPrincipal());
+                User user = this.getUserPersistence().getLockedUser(this.getPrincipal());
+                this.printUser(user);
+                this.systemOut.println("User Status: LOCKED");
+            }
```

Comment:
The status will be a nice addition

Reply:
I found that both active and pending accounts were displayed in --view, and maybe after practice one could discern between it. The other reason for this statement is to allow the admin to tell if an account is locked, or truly doesn't exist in the system. (the add of .getLockedUser() ) - before if a user was locked it was never displayed with --view.